opsdroid/core.py

```diff
@@ -92,7 +92,7 @@ def critical(self, error, code):
         """Exit due to unrecoverable error."""
         self.sys_status = code
         _LOGGER.critical(error)
-        print("Error: " + error)
+        print("Error:", error)
```

Comment:
Thanks for fixing this! I know it's only a tiny change but this probably should've been in a separate PR. I'll leave it for now, but something to bear in mind in the future.